tests/Unish/UnishTestCase.php

```diff
@@ -372,13 +377,56 @@ function fetchInstallDrupal($env = 'dev', $install = FALSE, $version_string = NU
       $this->drush('site-install', array($profile), $options);
       // Give us our write perms back.
       chmod($site, 0777);
+      chmod("$site/settings.php", 0777);
     }
     else {
       @mkdir($site);
       touch("$site/settings.php");
     }
   }
 
+  // pm-download cannot download from d6lts, so we will rough in our own download function
+  function downloadDrupal6($drupal_6_test_version, $root) {
```

Comment:
Have you tried [drush_download_file()](http://api.drush.org/api/drush/includes%21drush.inc/function/drush_download_file/8.0.x). That has optional caching. I'm fine with keeping this if you prefer.

Reply:
Is `drush_download_file()` available to Unish? Usually we avoid using Drush methods from the test framework. I was reluctant to break this pattern.

Reply:
Ah right. Makes sense.

IMO we can start breaking that pattern in Drush 9 but lets leave 8 alone.